.travis.yml

```diff
@@ -16,10 +16,10 @@ before_script:
   - sudo service mysql stop
   - sudo service postgresql stop
   - docker-compose up -d
-  - cp ormconfig.travis.json ormconfig.json
+  - npm run setup:config
```

Comment:
this is not needed really, previous command is simple and I would prefer not to use gulp too much.